redis/redis.go

```diff
@@ -187,3 +187,27 @@ type SlowLog struct {
 	// ClientName is the name set via the CLIENT SETNAME command (4.0 only).
 	ClientName string
 }
+
+// Latency represents a redis LATENCY LATEST.
+type Latency struct {
+	// Latency event type.
+	EventType string
```

Comment:
I liked this change, but then I tried to write the comment starting with `EventType` I couldn't make a good sentence without it stuttering, do you have anything or would this work?
```suggestion
	// Name of the latest latency spike event.
	Name string
```

Then I read the [redis LATENCY LATEST](https://redis.io/commands/latency-latest/) docs and it refers to the field as name
> Event name.
Unix timestamp of the latest latency spike for the event.
Latest event latency in millisecond.
All-time maximum latency for this event.

Reply:
Good points all around with REDIS docs being final arbiter here. Fully agree with a name being a better option.